pyproject.toml

```diff
@@ -46,19 +46,22 @@ exclude_lines = [
 [tool.tox]
 legacy_tox_ini = """
 [tox]
-isolated_build = True
 envlist = site
+isolated_build = True
 
 [testenv]
-sitepackages = true
+package = wheel
 deps=
-  pyflakes
   pytest-cov
   python-dbusmock
-commands =
-  python3 -m pyflakes src/systemd_ctypes test
```

Comment:
This is the "pyproject.toml: overhaul tox config, again" commit, and it does not document dropping pyflakes at all. Was that an accident?

Reply:
I merged this all into a mega-commit including introducing ruff and pycodestyle.